react/Text/Text.less

```diff
@@ -0,0 +1,31 @@
+@import (reference) "~seek-style-guide/theme";
+
+.root {
+  .standardTextResponsive();
+  padding-bottom: @grid-row-height;
+
+  .root {
+    position: relative;
```

Comment:
Nope, this is the first instance of it, and may be the last. In this case, it was introduced because nesting Text components is an easy mistake to make, with very subtle side effects, and debugging it is quite tricky.